mreg/tests.py

```diff
@@ -285,6 +288,22 @@ def setUp(self):
 
         self.ns_sample = NameServer(name='ns.example.org',
                                     ttl=300)
+    
+    @mock.patch('django.db.models.Model.save')
+    def test_update_serialno_handles_database_error(self, mock_save):
+        mock_save.side_effect = DatabaseError
+        self.zone_sample.update_serialno(force=True)
+        
+        mock_save.assert_called_once()
+
+        # Assert that the zone's serial number has not changed
+        self.assertEqual(self.zone_sample.serialno, self.zone_sample.serialno)
+        
+        # Assert that updated is still False
+        self.assertEqual(self.zone_sample.updated, False)
+        
+        # Assert that serialno_updated_at has not changed
+        self.assertEqual(self.zone_sample.serialno_updated_at, self.zone_sample.serialno_updated_at)
```

Comment:
Ditto as above.

Reply:
Resovled in https://github.com/unioslo/mreg/pull/506/commits/ba422e766083f6f216fffdddff177f5e91530209